tools/matrix-bot/logger.ts

```diff
@@ -22,24 +17,49 @@ export class MessagesLogger {
     if (!eventId) {
       return;
     }
-    if (!msg.includes("graypaper.fluffylabs.dev")) {
+    const gpLinks = findLinks(msg);
+    if (!gpLinks.length) {
       return;
     }
 
+    // if there are several links in the message, find the link with the latest version
+    let versionName: string | undefined;
+
+    if (gpLinks.length === 1) {
+      versionName = parseLink(gpLinks[0], this.meta)?.versionName;
+    } else {
+      versionName = gpLinks
+        .map((link) => parseLink(link, this.meta))
+        .filter(Boolean)
+        .map((link) => link?.versionName)
+        .sort()
```

Comment:
```suggestion
        .map((link) => link?.versionName)
        .sort()
```

Feels a bit brittle to compare lexicographically, since `0.9.1` will be bigger than `0.10.1`.

Can you explain the reasoning behind the logic to look for the link to the newest version? If it's there to only handle replies correctly, we might just take the last link posted in the message?

Reply:
good point about sorting.

as for why im looking for the link to the newest version, ive seen messages like this: https://matrix.to/#/!ddsEwXlCWnreEGuqXZ:polkadot.io/$ZhInZB-ZLyZbEGp-glzfvd3J_tHOK9GIXBTYFEA4eFw?via=polkadot.io&via=matrix.org&via=parity.io
i would say the note we would make from this message would be most relevant to the latest version rather than one of the old ones.

Reply:
Yup, makes sense!